index.js

```diff
@@ -427,8 +427,8 @@ function asStream(opts) {
 		proxy.emit('response', res);
 	});
 
+	ee.once('error', proxy.emit.bind(proxy, 'error'));
```

Comment:
`cacheable-request` could potentially emit more than one error. These should be passed through.